src/gui/progressbardelegate.cpp

```diff
@@ -65,9 +64,11 @@ void ProgressBarDelegate::paint(QPainter *painter, const QStyleOptionViewItem &o
     QStyleOptionProgressBar newopt;
     newopt.initFrom(&m_dummyProgressBar);
     newopt.rect = option.rect;
+    newopt.state = option.state;
```

Comment:
@jagannatharjun 
I hope this change won't introduce any regressions.

Reply:
I don't have KDE OS available, have you checked with a disabled file in the content tree view?

Reply:
>have you checked with a disabled file in the content tree view?

Good catch. The completed part in the progress bar is visible when selected or highlighted and just _barely_ visible otherwise in Breeze.
In Breeze dark the untouched is more visible.

![light2](https://user-images.githubusercontent.com/6451685/109671209-8dcb3c80-7b7c-11eb-92b6-c61b984e4f4e.JPG)![dark2](https://user-images.githubusercontent.com/6451685/109671222-915ec380-7b7c-11eb-97b8-d5f92d159db2.JPG)

Reply:
So I suppose it's fixed now?

Reply:
>So I suppose it's fixed now?

TBH I not familiar with the breeze theme and not sure what is considered to be normal... I would need others help on confirming it.

Reply:
>TBH I not familiar with the breeze theme

It's the default theme i think, that's why i mention it.

>So I suppose it's fixed now?

Nope. Look at the light theme pic 2nd to last line. It's unchecked and the filled part of the progress bar is almost invisible when it's untouched (not selected or highlighted).

Reply:
>It's unchecked and the filled part of the progress bar is almost invisible when it's untouched (not selected or highlighted).

That is the progress bar in disabled state, or is it wrong? Could you show an example of a correct one?

Reply:
>That is the progress bar in disabled state

The filled portion of the bar is almost invisible which should be visible either way IMO like in windows and even the last line in the Breeze dark pic. Anyway i now tested without this patch and it's exactly the same... someone made a poor choice of colors in Breeze i guess so forget what i said.

Reply:
OK, seems no more issues now. @glassez you may approve it.